packages/util/src/errors.ts

```diff
@@ -101,3 +101,9 @@ export class InvalidArgumentError extends CustomError {
     super(`Invalid argument '${argName}': ${message}`);
   }
 }
+
+export class HandleParsingError extends CustomError {
```

Comment:
This seems out of place, should be in `projection` package. I suggest defining it in the same file where it's thrown.

Reply:
Fixed on 6c65203